proto/prysm/v1alpha1/validator.proto

```diff
@@ -471,11 +471,9 @@ message DutiesRequest {
 }
 
 message DutiesResponse {
-    repeated Duty duties = 1 [deprecated = true];
+    repeated Duty current_epoch_duties = 1;
 
-    repeated Duty current_epoch_duties = 2;
-
-    repeated Duty next_epoch_duties = 3;
+    repeated Duty next_epoch_duties = 2;
```

Comment:
Do not change field numbers!!

See my suggestion in your other PR https://github.com/prysmaticlabs/prysm/pull/11248#discussion_r948328529